verification/curator-service/ui/src/components/new-case-form-fields/GenomeSequences.tsx

```diff
@@ -0,0 +1,104 @@
+                                        >
+                                            <DateField
+                                                name={`genomeSequences[${index}].sampleCollectionDate`}
+                                                label="Sample collection date"
+                                            ></DateField>
+                                            <Field
+                                                className={classes.field}
+                                                name={`genomeSequences[${index}].repositoryUrl`}
+                                                type="text"
+                                                label="Repository URL"
+                                                fullWidth
+                                                component={TextField}
+                                            ></Field>
+                                            <Field
+                                                className={classes.field}
+                                                name={`genomeSequences[${index}].sequenceId`}
+                                                type="text"
+                                                label="Sequence ID"
+                                                fullWidth
+                                                component={TextField}
+                                            ></Field>
+                                            <Field
+                                                className={classes.field}
+                                                name={`genomeSequences[${index}].sequenceName`}
+                                                type="text"
+                                                label="Sequence name"
+                                                fullWidth
+                                                component={TextField}
+                                            ></Field>
+                                            <Field
+                                                className={classes.field}
+                                                name={`genomeSequences[${index}].sequenceLength`}
+                                                type="number"
+                                                label="Sequence length"
+                                                fullWidth
+                                                component={TextField}
+                                            ></Field>
+                                            <Field
+                                                name={`genomeSequences[${index}].notes`}
+                                                type="text"
+                                                label="Notes"
+                                                multiline={true}
+                                                fullWidth
+                                                rows="3"
```

Comment:
you can remove that for now, I know I added that to the other notes but after talking to Shruu the right behavior for textareas is to start on one line and grow as needed, I am not sure how to implement that but I agree that's a better pattern than a fixed number of row.

Reply:
Done. It does grow when you hit enter.